src/routes/(console)/project-[region]-[project]/settings/usage/[[invoice]]/+page.svelte

```
@@ -30,6 +30,8 @@
         data.usage.buildsStorageTotal;
     $: imageTransformations = data.usage.imageTransformations;
     $: imageTransformationsTotal = data.usage.imageTransformationsTotal;
+    $: screenshotsGenerated = (data.usage as any).screenshotsGenerated ?? null;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)